actionpack/lib/action_controller/metal/mime_responds.rb

```diff
@@ -156,16 +156,16 @@ module MimeResponds
     # It works for both inline:
     #
     #   respond_to do |format|
-    #     format.html.any   { render text: "any"   }
-    #     format.html.phone { render text: "phone" }
+    #     format.html.any   { render body: "any"   }
+    #     format.html.phone { render body: "phone" }
     #   end
     #
     # and block syntax:
     #
     #   respond_to do |format|
     #     format.html do |variant|
-    #       variant.any(:tablet, :phablet){ render text: "any" }
-    #       variant.phone { render text: "phone" }
+    #       variant.any(:tablet, :phablet){ render body: "any" }
+    #       variant.phone { render body: "phone" }
```

Comment:
Would avoid using the `body:` option in docs. Makes sense to use `plain:` if we're rendering text.